SU-HSDO/suhumsci
# STN-719:  Traditional Color Band Refinement

# READY FOR REVIEW

## Summary
Add extra padding to the Traditional theme Color Band Component.

### Before
<img width="1345" alt="before" src="https://user-images.githubusercontent.com/12678977/121733776-ebe3b600-cac1-11eb-92da-5373bb6e8f3b.png">

### After
<img width="1344" alt="after" src="https://user-images.githubusercontent.com/12678977/121733802-f1410080-cac1-11eb-9b7a-d81c769ce468.png">

## Need Review By (Date)
6/15

## Urgency
medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. If you have a flexible page with a color band component, you may use that to test the work in this PR. If you have sync with staging, you can use http://sparkbox-sandbox.suhumsci.loc/qa-colorband. Otherwise, create a new flexible page and add a color band component and a mix of other content.
3. - [x] Confirm the padding for the **Traditional** theme color band component has increased.
4. - [x] Confirm the padding for the **Colorful** theme color band component has not changed.
